WalletWasabi.Gui/Controls/WalletExplorer/TransactionViewModel.cs

```diff
@@ -92,7 +92,7 @@ public double ClipboardNotificationOpacity
 			set => this.RaiseAndSetIfChanged(ref _clipboardNotificationOpacity, value);
 		}
 
-		public CancellationTokenSource CancelClipboardNotification { get; set; }
+		public CancellationTokenSource? CancelClipboardNotification { get; set; }
```

Comment:
It seems that this property is used only in this file. Can it be a local variable in `TryCopyTxIdToClipboardAsync` method?

Could you explain what it does? If it is a workaround, can it be possibly removed?

Reply:
I don't know what it does.

Here is where similar code exists:
https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Gui/Controls/MultiTextBox.cs#L60
https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Gui/ViewModels/AddressViewModel.cs#L213

Here is where that code was added https://github.com/zkSNACKs/WalletWasabi/commit/fcad0bc36e5c7fdada761b2b6c77ee38e57f19b3 and it is related to #1276

Maybe @nopara73 can clarify that.

Reply:
I had a look and .net core + clipboard support is non-trivial feat. 

That `CancelClipboardNotification` is used to prevent "fast" copying to clipboard. It seems.